lib/gears/shape.lua

```diff
@@ -195,3 +484,4 @@ function module.transform(shape)
 end
 
 return module
+-- kate: space-indent on; indent-width 4; replace-tabs on;
```

Comment:
Huh? This doesn't have a vim modeline (weird...) and we now have kate mode lines?

I guess everyone should somehow use the `.editconfig` file that we have in awesome, I just don't know how. :-(
(Adding special comments for every editor isn't going to fly.)